test/unit/lib/plugins/aws/package/compile/events/stream.test.js

```diff
@@ -1432,6 +1439,40 @@ describe('AwsCompileStreamEvents', () => {
           awsCompileStreamEvents.serverless.service.provider.compiledCloudFormationTemplate
             .Resources.FirstEventSourceMappingKinesisDef.Properties.MaximumRecordAgeInSeconds
         ).to.equal(undefined);
+
+        // event 9
+        expect(
```

Comment:
We should not expand the old tests, but instead we should write new ones with `runServerless` util: https://github.com/serverless/serverless/blob/master/test/README.md